repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -104,7 +95,6 @@ class ClassLoaderTest extends ReposeValveTest {
         then: "the request header should equal BAR"
         mc.handlings.size() == 1
         mc.receivedResponse.code == "200"
-        mc.handlings[0].request.headers.getFirstValue("FOO") == "BAR"
```

Comment:
Why remove this check? It seems like we do expect the "FOO" header to have a value of "BAR". Was this done because of the weird behavior you were seeing with Deproxy?

Also, removing this check makes the `then` comment inaccurate.